rules/windows/defense_evasion_disabling_windows_logs.toml

```diff
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2021/05/06"
+maturity = "development"
+updated_date = "2021/05/06"
+
+[rule]
+author = ["Elastic","Ivan Ninichuck","Austin Songer"]
+description = """
+Identifies attempts to disable EventLog and logman. This is often done by attackers in an attempt to evade detection on
```

Comment:
if the rule purpose is to cover only logman, suggest to adjust the description to `Identifies attempts to disable EventLog via logman Windows utility. ....`